### PR TITLE
Add error handling and remove global variable usage in utility hacks.

### PR DESCRIPTION
**Pull request by MelnCat**
## Description
This pr adds error handling to several hacks that had not been handled properly prior. Additionally, it also removes the usage of a global variable to reduce pollution in the window object.

### CODE REVIEW

File: willsCheatMenu/src/hacks/utility.ts [hunk 1]
```diff
@@ -86,15 +87,19 @@ new Hack(category.utility, "Edit walkspeed").setClick(async () => {
...
+
 new Toggler(category.utility, "Toggle Click Teleporting").setEnabled(async () => {
-	window.teleportingInterval = setInterval(() => {
+	teleportingInterval = setInterval(() => {
 		try {
 			_.player._playerContainer.walkSpeed = 500;
-		} catch (e) {}
+		} catch (e) { 
+			await Toast.fire("An error occurred.", `An error occurred while toggling click teleporting. Error: ${e}`, "error");
+		}
```
Comment (PatheticMustan) [inline willsCheatMenu/src/hacks/utility.ts:98]: this will spam toasts as fast as `setInterval` can speeeeed. :3

**Review by PatheticMustan — CHANGES_REQUESTED** [review]
if you're adding error handling, you'll need to add a check that you haven't sent a toast in the last few seconds, or at all. otherwise lgtm :3

File: willsCheatMenu/src/hacks/utility.ts (hunk 1 above)
Comment (MelnCat) [reply willsCheatMenu/src/hacks/utility.ts:98]: This is a feature. It will emphasize the importance of the error.r

**Review by PatheticMustan — APPROVED** [review]
hell yeah

**Pull request merged**